### PR TITLE
Updates LiveSync install section

### PR DESCRIPTION
**Pull request by franrob-projects**
This PR:

- Updates the overall structure of the doc.
- Clearly explains the purpose of the Database Connector
- Creates clear paths for users to follow with or without an existing database.
- Adds clearer language to describe the processes accurately.
- Improves the overall language.

[COL-657: Integration guidelines for existing systems with LiveSync](https://ably.atlassian.net/browse/COL-657)

### CODE REVIEW

File: content/livesync/connector/index.textile [hunk 1]
```diff
@@ -87,36 +97,37 @@ ADBC_NODES_TABLE_AUTO_CREATE=true
...
-The response should be similar to the following:
+The response should look similar to the following:
 
 ```[json]
 {"ably":{"status":"up"},"nodes_table":{"status":"up"},"outbox_table":{"status":"up"},"postgres":{"status":"up"}}
 ```
+h3(#varify-docker). Varify connection using Docker 
 
-With a PostgreSQL database and Database Connector instance running, the next step is to write a record to the outbox table and confirm it has been published over Ably:
+Varify your connection to the Ably network now you have a PostgreSQL database and Database Connector instance running. The following examples show how to write a record to the outbox table and confirm its publication.
```
Comment (GregHolmes) [inline content/livesync/connector/index.textile:122]:
```suggestion
Verify your connection to the Ably network now you have a PostgreSQL database and Database Connector instance running. The following examples show how to write a record to the outbox table and confirm its publication. 
```

File: content/livesync/connector/index.textile [hunk 2]
```diff
@@ -87,36 +97,37 @@ ADBC_NODES_TABLE_AUTO_CREATE=true
...
 ```[json]
 {"ably":{"status":"up"},"nodes_table":{"status":"up"},"outbox_table":{"status":"up"},"postgres":{"status":"up"}}
 ```
+h3(#varify-docker). Varify connection using Docker 
 
-With a PostgreSQL database and Database Connector instance running, the next step is to write a record to the outbox table and confirm it has been published over Ably:
+Varify your connection to the Ably network now you have a PostgreSQL database and Database Connector instance running. The following examples show how to write a record to the outbox table and confirm its publication. 
 
-Open a terminal and subscribe to a "channel":/channels named @foo@ using "Server-Sent Events":/protocols/sse
+Subscribe to a "channel":/channels named @foo@ using "Server-Sent Events:":/protocols/sse
```
Comment (GregHolmes) [inline content/livesync/connector/index.textile:124]: Is it important to tell them what to name their channel?
I realise it's used elsewhere on the page. I'm not sure.

File: content/livesync/connector/index.textile [hunk 3]
```diff
@@ -199,15 +218,17 @@ ADBC_NODES_TABLE_AUTO_CREATE=true
 
 Railway will restart the @adbc-test@ service with the newly applied configuration.
 
-With a PostgreSQL database and Database Connector instance running on Railway, the next step is to write a record to the outbox table and confirm it has been published over Ably:
+h3(#varify-railway). Varify connection using Railway
```
Comment (GregHolmes) [inline content/livesync/connector/index.textile:221]:
```suggestion
h3(#verify-railway). Verify connection using Railway
```

File: content/livesync/connector/index.textile [hunk 4]
```diff
@@ -199,15 +218,17 @@ ADBC_NODES_TABLE_AUTO_CREATE=true
...
 
-Open a terminal and subscribe to a "channel":/channels named @foo@ using "Server-Sent Events":/protocols/sse
+Varify your connection to the Ably network now you have a PostgreSQL database and Database Connector instance running. The following examples show how to write a record to the outbox table and confirm its publication.
```
Comment (GregHolmes) [inline content/livesync/connector/index.textile:223]:
```suggestion
Verify your connection to the Ably network now you have a PostgreSQL database and Database Connector instance running. The following examples show how to write a record to the outbox table and confirm its publication. 
```

**Review by GregHolmes — CHANGES_REQUESTED** [review]
A couple spelling mistakes need checking out first.

File: content/livesync/connector/index.textile (hunk 4 above)
Comment (franrob-projects) [reply content/livesync/connector/index.textile:223]: Thanks, Greg. I was convinced it was "varify" lol

**Review by GregHolmes — APPROVED** [review]

File: content/livesync/connector/index.textile [hunk 5]
```diff
@@ -4,11 +4,17 @@ meta_description: "The Ably Database Connector is available as a Docker containe
 product: livesync
 ---
 
-The Database Connector uses the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html to broadcast application-specific change events from your database to frontend clients via Ably "channels":/channels.
+The Ably Database Connector (ADBC) sends updates from your database to frontend clients through Ably "channels":/channels using the "outbox pattern":https://docs.aws.amazon.com/prescriptive-guidance/latest/cloud-design-patterns/transactional-outbox.html. Whenever you update data in your database, this system records changes in an outbox table. The Database Connector detects these changes and publishes them as messages to specified channels. Client applications using the "Models SDK":/livesync/models/models subscribe to these channels to receive updates and refreshes to their local data.
```
Comment (mschristensen) [inline content/livesync/connector/index.textile:7]: "this system" to me implies that the system does it for you automatically, but that is not the case, the developer has to record the change in the outbox table

Comment (mschristensen) [reply content/livesync/connector/index.textile:7]: Also maybe remove "refreshes"?

File: content/livesync/connector/index.textile [hunk 6]
```diff
@@ -20,11 +26,13 @@ docker pull ghcr.io/ably-labs/adbc:latest
 <p>Please "reach out":https://docs.google.com/forms/d/e/1FAIpQLSd00n1uxgXWPGvMjKwMVL1UDhFKMeh3bSrP52j9AfXifoU-Pg/viewform if you have any feedback or prefer to use an Ably-hosted version of LiveSync.</p>
 </aside>
 
-h2(#docker). Run using Docker Compose
+h2(#existing-db). Run an existing database
+
+Use "Docker Compose":https://docs.docker.com/compose/ to run an exisiting "PostgreSQL":https://www.postgresql.org/ with the Database Connector.
```
Comment (mschristensen) [inline content/livesync/connector/index.textile:31]: typo on `existing`

Also the docker compose creates a pg instance for you by default

File: content/livesync/connector/index.textile [hunk 7]
```diff
@@ -87,36 +97,37 @@ ADBC_NODES_TABLE_AUTO_CREATE=true
 The @docker-compose.yml@ will load this configuration into the @adbc@ container as environment variables.
 
 <aside data-type='important'>
-<p>If using your own API key, ensure that it has the @publish@ "capability":/auth/capabilities on all channels that you intend to publish to.</p>
+<p>If using an API key from a provider other than Ably, verify it has the @publish@ "capability":/auth/capabilities on all channels you intend to publish to. </p>
```
Comment (mschristensen) [inline content/livesync/connector/index.textile:100]: This isn't quite right, only Ably can issue API keys to access Ably. Perhaps just "Ensure your API key has the publish capability ..."

File: content/livesync/connector/index.textile [hunk 8]
```diff
@@ -320,15 +317,50 @@ services:
      - ./adbc.yaml:/adbc.yaml:ro # mount yaml config file
 ```
 
-h4(#command). Command line flags
+h3(#command). Command line flags
 
-To specify configuration options using command line flags:
+Specify configuration options usinf CLI:
```
Comment (mschristensen) [inline content/livesync/connector/index.textile:322]: typo

File: content/livesync/connector/index.textile (hunk 5 above)
Comment (franrob-projects) [reply content/livesync/connector/index.textile:7]: 👍

File: content/livesync/connector/index.textile (hunk 6 above)
Comment (franrob-projects) [reply content/livesync/connector/index.textile:31]: 👍

File: content/livesync/connector/index.textile (hunk 7 above)
Comment (franrob-projects) [reply content/livesync/connector/index.textile:100]: 👍

File: content/livesync/connector/index.textile (hunk 8 above)
Comment (franrob-projects) [reply content/livesync/connector/index.textile:322]: 👍

**Pull request merged**